npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -193,19 +191,33 @@ func (pMgr *PolicyManager) bootup(_ []string) error {
 	}
 
 	// 3. add/reposition the jump to AZURE-NPM
-	if err := pMgr.positionAzureChainJumpRule(); err != nil {
+	if err := pMgr.positionAzureChainJumpRule(util.IptablesForwardChain); err != nil {
 		baseErrString := "failed to add/reposition jump from FORWARD chain to AZURE-NPM chain"
 		metrics.SendErrorLogAndMetric(util.IptmID, "error: %s with error: %s", baseErrString, err.Error())
 		return npmerrors.SimpleErrorWrapper(baseErrString, err) // we used to ignore this error in v1
 	}
+
+	// 4. add the jump to AZURE-NPM from OUTPUT chain
+	if err := pMgr.positionAzureChainJumpRule(util.IptablesOutputChain); err != nil {
+		baseErrString := "failed to add/reposition jump from OUTPUT chain to AZURE-NPM chain"
+		metrics.SendErrorLogAndMetric(util.IptmID, "error: %s with error: %s", baseErrString, err.Error())
+		return npmerrors.SimpleErrorWrapper(baseErrString, err) // we used to ignore this error in v1
+	}
 	return nil
 }
 
 // reconcile does the following:
 // - creates the jump rule from FORWARD chain to AZURE-NPM chain (if it does not exist) and makes sure it's after the jumps to KUBE-FORWARD & KUBE-SERVICES chains (if they exist).
+// - creates the jump rule from OUTPUT chain to AZURE-NPM chain (if it does not exist).
 // - cleans up stale policy chains. It can be forced to stop this process if reconcileManager.forceLock() is called.
 func (pMgr *PolicyManager) reconcile() {
-	if err := pMgr.positionAzureChainJumpRule(); err != nil {
+	if err := pMgr.positionAzureChainJumpRule(util.IptablesForwardChain); err != nil {
+		msg := fmt.Sprintf("failed to reconcile jump rule to Azure-NPM due to %s", err.Error())
```

Comment:
could we say which chain in the error msg (one right below too)

Reply:
updated